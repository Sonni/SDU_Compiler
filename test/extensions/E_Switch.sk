var a: int;
var b: int;

a = 40;
b = a;

# Switch with breaks;

switch (a) {
case 40:
    {
    write 30;
    break;
    }
case b:
    {
        write 1;
        b = 42;
        break;
    }
case 10:
    {
    write 11;
    break;
    }
}

write b;

# Switch without breaks


switch (b) {
case 1:
    write 1;
case 40:
    write 2;
case 10:
    {
    write 3;
    }
}