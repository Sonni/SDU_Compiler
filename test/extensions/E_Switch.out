30
40
2
3
